src/app/students/browsecredits/credits.json

```diff
@@ -0,0 +1,62 @@
+[
```

Comment:
where is this be pulled into? what component?

Reply:
Removed this json file to avoid confusion. json data (dummy data) now located within CreditList component. This is now resolved. Thanks!